pkg/helm/release/manager.go

```diff
@@ -174,10 +174,11 @@ func (m manager) InstallRelease(ctx context.Context) (*rpb.Release, error) {
 			// Only log a message about a rollback failure if the failure was caused
 			// by something other than the release not being found.
 			if uninstallErr != nil && !notFoundErr(uninstallErr) {
+				//TODO: What to change this to?
 				return nil, fmt.Errorf("failed installation (%s) and failed rollback (%s)", err, uninstallErr)
```

Comment:
```suggestion
				return nil, fmt.Errorf("failed installation (%w) and failed rollback (%w)", err, uninstallErr)
```


`%w` works anywhere in the formatting string.

Reply:
Per [Errorf reference](https://golang.org/pkg/fmt/#Errorf), and re-iterated in this [issue](https://github.com/golang/go/issues/34060), 
**It is invalid to include more than one %w verb**, so what should the change be?

Reply:
Use `%v` for `err` and `%w` for `uninstallErr` since we’re returning after checking the latter.

Reply:
Done.